test/core/core_test.exs

```diff
@@ -0,0 +1,40 @@
+# Copyright(c) 2015-2021 ACCESS CO., LTD. All rights reserved.
+
+defmodule AntikytheraCoreTest do
+  use Croma.TestCase
+  alias AntikytheraEal.ClusterConfiguration
+
+  setup do
+    :meck.new(ClusterConfiguration, [:passthrough])
+
+    on_exit(fn ->
+      :meck.unload()
+    end)
```

Comment:
```suggestion
    on_exit(&:meck.unload/0)
```

Reply:
Fixed in https://github.com/access-company/antikythera/pull/168/commits/a8f193fcb6aa30936db2813e70cdf84efbdc929b